crates/cybercmd/src/config/game_config_list.rs

```diff
@@ -0,0 +1,75 @@
+use std::{ffi::OsStr, fs};
+
+use common::extensions::PathExt;
+use uniquote::Quote;
+
+use super::{app_context::Paths, GameConfig};
+
+pub struct GameConfigList(Vec<GameConfig>);
+
+impl<T> core::ops::Index<T> for GameConfigList
+where
+    Vec<GameConfig>: core::ops::Index<T>,
+{
+    type Output = <Vec<GameConfig> as core::ops::Index<T>>::Output;
+
+    #[inline]
+    fn index(&self, idx: T) -> &Self::Output {
+        <Vec<GameConfig> as core::ops::Index<T>>::index(&self.0, idx)
+    }
+}
+
+impl IntoIterator for GameConfigList {
+    type IntoIter = <Vec<GameConfig> as IntoIterator>::IntoIter;
+    type Item = <Vec<GameConfig> as IntoIterator>::Item;
+
+    #[inline]
+    fn into_iter(self) -> Self::IntoIter {
+        <Vec<GameConfig> as IntoIterator>::into_iter(self.0)
```

Comment:
```suggestion
        self.0.into_iter()
```

Reply:
Oh, I forgot to clean those up. I generated the `IntoIterator` implementation using `rustc -Zunpretty`. It produced inverted versions of all the `self.method` calls.

Reply:
interesting, it did look like code generated from macros, the ultra-verbose syntax is a bit of a overkill there because the into_iter should always be unambiguous here

Reply:
UFCS! Universal Function Call Syntax. That's the name of it. That versus method calls.